lib/pages/bluehost/bluehost-login-page.js

```diff
@@ -0,0 +1,19 @@
+import { By as by } from 'selenium-webdriver';
+import config from 'config';
+import BaseContainer from '../../base-container';
+import * as driverHelper from '../../driver-helper';
+
+export default class BluehostLoginPage extends BaseContainer {
+	constructor( driver, visit = true ) {
+		const url = config.get( 'bluehostLoginUrl' );
+		super( driver, by.css( 'form.login_form' ), visit, url );
+	}
+
+	logon() {
+		const username = config.get( 'bluehostUsername' );
```

Comment:
I'm torn on the username/password config options.  They could either go inside the big `bluehostConfig` parameter I suggested in another comment, or inside the `testAccounts` object with the other username/passwords.

If this is just a standalone test it probably makes more sense to keep them with the other Bluehost parameters.  But if we expand to other tests that involve logging in here then we'd probably want to put something in the LoginFlow object, in which case it might make more sense to put them in the `testAccounts`.

But we can burn that bridge when we get there :)